src/community/ogcapi/dggs/dggs-core/src/main/java/org/geotools/dggs/rhealpix/RHealPixDGGSFactory.java

```diff
@@ -46,7 +46,7 @@ public Parameter[] getParametersInfo() {
 
     @Override
     public DGGSInstance createInstance(Map<String, ?> params) throws IOException {
-        return new RHealPixDGGSInstance(new JEPWebRuntime(INITIALIZER), "TB16-Pix");
+        return new RHealPixDGGSInstance(new JEPWebRuntime(INITIALIZER), "rHEALPix");
```

Comment:
Class Javadoc reference to TB16-Pix needs to be updated

Reply:
Done